openedx/core/djangoapps/courseware_api/views.py

```diff
@@ -135,8 +135,7 @@ def content_type_gating_enabled(self):
 
     @property
     def can_show_upgrade_sock(self):
-        return (DISPLAY_COURSE_SOCK_FLAG.is_enabled(self.course_key) and
-                can_show_verified_upgrade(self.effective_user, self.enrollment_object))
+        return DISPLAY_COURSE_SOCK_FLAG.is_enabled(self.course_key)
```

Comment:
Anywhere the MFE checks if it can show the upgrade sock, it also checks if `verifiedMode` is provided, which is where we now check if we can show the verified upgrade info (to make it more generic, since non-sock locations now care about that).